src/data/image.ts

```diff
@@ -2,6 +2,14 @@ import sharp from "sharp";
 import S3 from "aws-sdk/clients/s3";
 import stream from "stream";
 import { google } from "googleapis";
+import memoize from "p-memoize";
```

Comment:
I'm using this memoize library because it gives the ability to add a timeout which I find incredibly handy.